### PR TITLE
Added functions to store values into arrays

### PR DESCRIPTION
**Pull request by RuslanSinyavsky**

### CODE REVIEW

File: coen320_main [hunk 1]
```diff
@@ -0,0 +1,389 @@
+// TODO: Implement a second array, and test with it
```
Comment (ginywiny) [inline coen320_main]: Why are you repushing this?

File: queryMethods.cpp [hunk 2]
```diff
@@ -20,49 +28,181 @@ void delay(int number_of_seconds)
...
+    int countVal = 0;
+    int rowNumberInd;
+    while (fgets(line, 1024, stream))
+    {        
+        char* tmp = strdup(line);
+        double desiredvalue;
+        desiredvalue = atof(getfield(tmp, 1, 1));
+        //printf("My field is col 3: %s\n", getfield(tmp,1 , 1));
+        if (countVal == rowNumberInd) {
+            printf("Hope it Works this time %f", desiredvalue);
```
Comment (ginywiny) [inline queryMethods.cpp:256]: Does it work though?

File: coen320_main (hunk 1 above)
Comment (RuslanSinyavsky) [reply coen320_main]: When i updated my branch i guess it did soemthing

File: queryMethods.cpp (hunk 2 above)
Comment (RuslanSinyavsky) [reply queryMethods.cpp:256]: from what i tested it gave me the right values, dont need it tho, its just incase you would need it so we dont have to remake it

File: coen320_main (hunk 1 above)
Comment (ginywiny) [reply coen320_main]: You added all files into a commit, thats why. Please untrack this, I don't want merge conflicts on the main file.

Comment (RuslanSinyavsky) [reply coen320_main]: oh well, still the same file tho so...

File: queryMethods.cpp (hunk 2 above)
Comment (ginywiny) [reply queryMethods.cpp:200]: Can you move these query methods into a new file.

File: queryMethods.cpp [hunk 3]
```diff
@@ -20,49 +28,181 @@ void delay(int number_of_seconds)
...
+    creatArrayOfEngineCoolantTemperatureValues();
+    creatArrayOfCurrentGearValues();
+    creatArrayOfTransmissionOilTemperatureValues();
+    creatArrayOfVehicleSpeedValues();
+    creatArrayOfAccelerationSpeedLongitudinalValues();
+    creatArrayOfIndicationOfBreakSwitchValues();
+
+    for (int i = 0; i < 20; i++) {
+        printf("Main output FC %f \n", fuelConsumption[i]);
+        delay(1);
```
Comment (ginywiny) [inline queryMethods.cpp]: Don't use delay(1), use sleep(1).

**Pull request merged**